tables/copy-zenodo-tables.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# S. Spielman for CCDL, 2023
+# This script copies figure data CSV files that were exported
+#  within individual modules into the `zenodo-upload/` directory here.
+#  Context: https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1692
+
+# enviroment settings
+set -eo pipefail
+
+# Find current directory based on this script
+WORKDIR=$(dirname "${BASH_SOURCE[0]}")
+cd "$WORKDIR"
+
+# Get base directory of project
+cd ..
+BASEDIR="$pwd"
+cd -
```

Comment:
It failed because of the earlier `cd`... 
```suggestion
# Find current directory based on this script
WORKDIR="$(dirname "${BASH_SOURCE[0]}")"

# Get base directory of project
BASEDIR="$(dirname "$WORKDIR")"
```